pkg/gatherers/conditional/gathering_rule.schema.json

```diff
@@ -217,6 +217,86 @@
                             "pattern": "^[a-zA-Z0-9_]{1,128}$"
                         }
                     }
+                },
+                "^pod_logs$": {
+                    "type": "object",
+                    "title": "GatherPodLogsParams",
+                    "required": [
+                        "resource_filter"
```

Comment:
Shouldn't the `log_filter` be required as well or at least the `message_to_search` should be required IMO

Reply:
Maybe? I'm trying to remember if we have some log gatherer that doesn't filter the errors. 🤔  But, I'll check.

Reply:
(I'm double checking it, but probably no problem to leave it as required for conditions)

Reply:
I'll move this to the epic. I think that we will need to validate the schema definition with the new requirements.